com.unity.render-pipelines.universal/ShaderLibrary/UnityInput.hlsl

```diff
@@ -153,7 +153,11 @@ float4 unity_MotionVectorsParams;
 CBUFFER_END
 
 #if defined(USING_STEREO_MATRICES)
+#if defined(USING_BUILTIN_STEREO_MATRICES) && (defined(SHADER_API_D3D11) || defined(SHADER_API_D3D12))
```

Comment:
I am curious about the DX11 and DX12 logic here. Should we apply the logic for all gfx APIs?

Reply:
When I ran my tests, I was only able to repro the problem on DirectX APIs (with Oculus link or MockHMD), Vulkan PC worked normally (with MockHMD). On the Quest it was working, so I targeted that change for DX only. 

I'll re-run some tests later today to confirm everything still works on Quest.

Reply:
I am curious about how does it work on Vulkan but no DX11/DX12. Should we look into that? I think in SRP land, we should keep logic the same for all gfx APIs if possible.